src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/Progress/ProgressRing_Active.xaml.cs

```diff
@@ -14,17 +14,15 @@
 using Windows.UI.Xaml.Media;
 using Windows.UI.Xaml.Navigation;
 
-// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
-
-namespace SamplesApp.Samples.Progress
+namespace UITests.Shared.Windows_UI_Xaml_Controls.Progress
 {
 	/// <summary>
-	/// An empty page that can be used on its own or navigated to within a Frame.
+	/// ignoreInAutomatedTests set to true because progressrings animate.
```

Comment:
Wondering if folks have any suggestions on how to make progressrings compatible with screenshot testing?

Reply:
This is actually a test that should be automated. For now iOS still fails to run automatically, but you can script it already for Android/Wasm.